modules/PropTypes.js

```diff
@@ -1,48 +1,50 @@
 import { PropTypes } from 'react'
+const {func, string, any, oneOf, shape} = PropTypes
```

Comment:
if we're gonna nit on this stuff I suppose we should put it in the eslint config.